probatus/feature_elimination/early_stopping_feature_elimination.py

```diff
@@ -167,6 +164,14 @@ def __init__(
                 and [LightGBM](https://lightgbm.readthedocs.io/en/latest/Parameters.html#metric-parameters).
                 Note that `eval_metric` is an argument of the model's fit method and it is different from `scoring`.
         """  # noqa
+        # TODO: This deprecation warning will removed when it's decided that this class can be deleted.
+        warnings.warn(
+            "The separate EarlyStoppingShapRFECV class is going to be deprecated"
+            " in a later version of Probatus, since its now part of the"
+            " ShapRFECV class. Please adjust your imported class name from"
+            " 'EarlyStoppingShapRFECV' to 'ShapRFECV'."
+        )
```

Comment:
[`warnings.warn`](https://docs.python.org/3/library/warnings.html#warnings.warn) allows you to provide a Warning class. It could be nice adding a `DeprecationWarning` here, so users could filter them if they wish :)